ftplugin/python/vim_ipython.py

```diff
@@ -142,7 +150,11 @@ def km_from_string(s=''):
                 p = p.lstrip().rstrip() # profile part of the string
                 fullpath = find_connection_file(k,p)
             else:
-                fullpath = find_connection_file(s.lstrip().rstrip())
+                s = s.lstrip().rstrip()
+                if len(s) == 0:
+                    fullpath = find_connection_file()
+                else:
+                    fullpath = find_connection_file(s.lstrip().rstrip())
```

Comment:
I guess it's not necessary to call s.lstrip().rstrip() again as It's already been done in line 153